c/httpfileservice.c

```diff
@@ -197,6 +197,27 @@ void deleteUnixDirectoryAndRespond(HttpResponse *response, char *absolutePath) {
   }
 }
 
+void createFileFromUnixDirectoryAndRespond(HttpResponse *response, char *absolutePath) {
```

Comment:
The function doesn't really create a file from a directory, it archives that directory.

Reply:
I see it's marked resolved. Can you comment on that?

Reply:
@ifakhrutdinov  so the name of the function should be changed right ?